### DISCUSSION TITLE
how to get more overlap flow

### DISCUSSION THREAD

**Discussion by Mario-vminion** [OrcaSlicer/OrcaSlicer#173] [Ideas]
hi looks very good ...first print was really good on the minion...
will there be an option to ajust the flow for the first layer and the infill to the wall...

**Comment by SoftFever** [comment]
Glad to hear it worked well :)
Yes, there is an option to adjust the flow of the first layer. You can find it in the quality tab.

Can you please clarify your second question?

**Reply by Mario-vminion** [reply]
Oke thanks i will look for that
